src/ServiceInsight.Licensing/InternalVisibleTo.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ServiceInsight")]
```

Comment:
Why is this necessary? Is this just to prevent us from making the licensing bits public? Is that as important in a product?

Reply:
This is important because we didn't want to make some of our licensing code public. We can't just make part of it public in SI codebase either, since it relies on some other 'internal' bits from the Licensing packages.